making_metadata_changes.md

```diff
@@ -0,0 +1,85 @@
+# How to make metadata changes
+
+## Introduction
+
+These steps outline how to edit the metadata for the phone number library to fix
+problems with validation or formatting.
+
+Note that we are currently not accepting validation or formatting pull requests,
+but you may use these guidelines to create changes in your own fork until we fix
+issues upstream.
+
+You can read more about the pull request [contribution
+guidelines](http://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md#pull-requests)
+and file an issue if we should make validation fixes!
+
+## Details
+
+### Edit the data
+
+Edit the appropriate files:
+
+*   `resources/PhoneNumberMetadata.xml` for normal validation / formatting
+    issues
+*   `resources/ShortNumberMetadata.xml` for short-code data
+*   `resources/PhoneNumberAlternateFormats.xml` for alternate patterns for phone
+    number matching
+*   `resources/geocoding/xx` for geocoding data (where `xx` is the language code
+    you wish to edit)
+*   `resources/carrier/xx` for carrier mapping data
+
+Note:
+
+*   Include sources for data where appropriate
```

Comment:
I don't think it should be there - this was from the existing wiki page, and applies only to PRs of metadata changes in xml files, which we don't do PRs for now.

I re-worded. I want to keep the idea of it in this doc, in case we bring back xml PRs, this point should be there.